malasakit-django/locale/tl/LC_MESSAGES/django.po

```diff
@@ -118,28 +126,28 @@ msgid ""
 "Research Institutes Project."
 msgstr ""
 "Isang proyekto ng CITRIS Connected Communities Initiative ng UC Berkeley, at "
-"ng Commission on Higher Education ng Pilipinas sa pamamagitan ng Philippine-"
-"California Advanced Research Institutes Project"
+"ng National University sa pamamagitan ng Philippine-California Advanced Research "
+"Institutes Project"
 
 #: pcari/templates/end.html:5 pcari/templates/end.html:6
 msgid "Thank You"
-msgstr ""
+msgstr "Maraming Salamat"
 
 #: pcari/templates/end.html:10
 msgid "We appreciate your participation in the Malasakit program."
-msgstr ""
+msgstr "Pinahahalagahan namin ang iyong pagmaMALASAKIT."
 
 #: pcari/templates/landing.html:6
 msgid "Welcome"
-msgstr ""
+msgstr "Maligayang pagdating"
 
 #: pcari/templates/landing.html:19
 #, python-format
 msgid ""
 "How prepared are you in times of disaster? %(num_responses)s barangay "
 "citizens have responded. Join them!"
 msgstr ""
-"Gaano ka kahanda sa panahon ng kalamidad? %(num_responses)s na ang nag "
+"Gaano ka kahanda sa panahon ng kalamidad? %(num_responses) na ang nag"
```

Comment:
Please add `s` back to the end of the format specifier `%(num_responses)`, just like for the English text (the `s` stands for a "string" placeholder).